src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/matcher/FilterableResolver.java

```diff
@@ -101,6 +103,16 @@ public FilterableResolver() {
   }
 
   protected List<DBFilterBase> resolve(CommandContext commandContext, T command) {
+    List<DBFilterBase> filter = matchRules.apply(commandContext, command);
+    if (filter.size() > docLimits.maxObjectProperties()) {
+      throw new JsonApiException(
+          ErrorCode.FILTER_FIELDS_LIMIT_VIOLATION,
+          String.format(
+              "%s: number of fields in filter has (%d) exceeds maximum allowed (%s)",
```

Comment:
"number of fields in filter has (%d) exceeds" -> "filter has %d fields, exceeds"